include/openthread/history_tracker.h

```diff
@@ -38,18 +38,12 @@ extern "C" {
 /**
  * @addtogroup api-history-tracker
  *
- * @{
- *
- */
-
-/**
  * @brief
- *   This header defines the public API for History Tracker.
+ *   Records the history of different events, for example RX and TX messages or network info changes. All tracked entries are timestamped.
```

Comment:
style: we limit line length to 120 characters
```suggestion
 *   Records the history of different events, for example RX and TX messages or network info changes. All tracked
 *   entries are timestamped.
```